neo/Network/P2P/Payloads/VersionPayload.cs

```diff
@@ -1,28 +1,30 @@
-﻿using Neo.IO;
-using System;
+﻿using System;
 using System.IO;
+using Neo.IO;
 
 namespace Neo.Network.P2P.Payloads
 {
     public class VersionPayload : ISerializable
     {
+        public uint Magic;
         public uint Version;
-        public ulong Services;
+        public VersionServices Services;
         public uint Timestamp;
         public ushort Port;
         public uint Nonce;
         public string UserAgent;
         public uint StartHeight;
         public bool Relay;
```

Comment:
We can remove this field, and use `VersionServices.NodeNetwork` as a replacement.